modules/data-wrangling/index.qmd

```diff
@@ -165,11 +168,301 @@ For the expression field, there are many functions and operators that are useful
 * `between()`, `near()`
 
 ```{r}
+#| eval: false
+penguins |>
+    filter(species = "Gentoo")
+```
+
+```{r}
+#| echo: false
+penguins |>
+    filter(species == "Gentoo") |>
+    head()
+```
+
+
+## arrange()
+
+It is for sorting your data based on one or more variables. By default, it will arrange your data in ascending order. If you want to sort by descending order, it can be done using the `desc()` function.
+
+```{r}
+#| eval: false
+penguins |>
+    arrange(bill_length_mm)
+
+penguins |>
+    arrange(desc(bill_length_mm))
+```
+
+```{r}
+#| echo: false
+penguins |>
+    arrange(bill_length_mm) |>
+    head()
+
+penguins |>
+    arrange(desc(bill_length_mm)) |>
+    head()
+```
+
+::: {.callout-tip}
+## Did you know
+
+What do you think will happen if you sort the penguin's data by the species where it is a character variable? If you guess that it will sort in **alphabetical order**, you are correct!!!
```

Comment:
More formal writing is preferred.

Reply:
Another common confusion is the sorting of numbers stored as text, for example `c("1", "3", "10", "14")`.
Although this might belong better elsewhere (basics of R objects maybe?)